outlook/src/taskpane/components/Contact/ContactList/ContactListItem/ContactListItem.tsx

```diff
@@ -1,71 +1,81 @@
+        this.props.onItemClick(this.props.partner);
     };
 
     render() {
-        const profileCardData = {
-            domain: undefined,
-            name: this.props.partner.name,
-            email: this.props.partner.email,
-            icon: this.props.partner.image ? "data:image;base64, " + this.props.partner.image : undefined,
-            initials: this.props.partner.getInitials(),
-            job: this.props.partner.title,
-            phone: this.props.partner.phone,
-            description: undefined,
-            twitter: undefined,
-            facebook: undefined,
-            crunchbase: undefined,
-            linkedin: undefined,
-            isBig: !this.context.isConnected() || (!this.props.partner.isAddedToDatabase()),
-            isCompany: this.props.partner.isCompany,
-        } as ProfileCardProps;
+        const logButton = this.props.partner.isAddedToDatabase() && (
+            <Logger resId={this.props.partner.id} model="res.partner" tooltipContent={_t('Log Email Into Contact')} />
+        );
 
-        let addButton = null;
-        let logButton = null;
+        const { name, email, title: jobTitle, phone } = this.props.partner;
 
-        if (this.props.partner.isAddedToDatabase())
-        {
-            logButton = (<Logger resId={this.props.partner.id} model="res.partner" tooltipContent={_t("Log Email Into Contact")}/>);
-        }
+        const emailDiv = email && (
+            <div className="contact-email">
+                <FontAwesomeIcon icon={faEnvelope} className="contact-info-icon fa-fw" />
+                <div className="text-truncate">
+                    <TooltipHost content={email} overflowMode={TooltipOverflowMode.Parent}>
+                        {email}
+                    </TooltipHost>
+                </div>
+            </div>
+        );
 
-        let classNames = "contact-list-item-container";
-        if (this.props.selectable)
-            classNames+= " contact-list-item-container-selectable";
+        const phoneDiv = phone && (
+            <div className="contact-phone">
+                <FontAwesomeIcon icon={faPhone} className="contact-info-icon fa-fw" />
+                <a className="link-like-button" href={`tel:${phone}`}>
+                    {phone}
+                </a>
+            </div>
+        );
 
-        return <React.Fragment>
-            <div onClick={this.onPartnerClick} className={classNames}>
-                {addButton}
-                <div className='contact-profile-card-container'>
-                    <ProfileCard {...profileCardData} />
+        return (
+            <div className={`contact-container ${this.props.onItemClick && 'clickable'}`} onClick={this.onPartnerClick}>
+                <div className="contact-card">
+                    {this.getIconOrInitials()}
+                    <div className="contact-info">
+                        <div className="contact-name">{jobTitle ? name + ', ' + jobTitle : name}</div>
```

Comment:
Can't you build the contact_name previously ? That way we have the logic before the final rendering (like emailDiv, phoneDiv).